scripts/drtprod

```diff
@@ -128,17 +128,21 @@ EOF"
     fi
     case "${2}" in
       "drt-main")
```

Comment:
nit: Change this to `drt-large` too?

Reply:
yep, was about to double check this before moving it out of draft, thanks for pointing it out!